concordium-consensus/src/Concordium/GlobalState/Persistent/Trie.hs

```diff
@@ -461,6 +462,41 @@ data TrieN fix k v
     | TrieN !Int !(fix (TrieF k v))
     -- ^A non empty trie with its size
 
+-- TODO: Fix and generalize
```

Comment:
todo?

Reply:
Well **some** version in the past needed it :) Removed now.